prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log('languages array', languages);
+
+var firstElement = languages[0];
+console.log('The first element of the array is', firstElement);
+
+var thirdElement = languages[2];
+console.log('The third element of the array is', thirdElement);
+
+var length = languages.length;
+console.log('The length of the array is', length);
+
+var lastElement = languages[5];
+console.log('The last element of the array is', lastElement);
```

Comment:
You can get the last element of any array by subtracting one from the array's length

```js
var lastElement = languages[length - 1];
```